inc/az_ulib_capability_api.h

```diff
@@ -123,25 +124,73 @@ typedef az_result (*az_ulib_capability_cancellation_callback)(
 typedef az_result (
     *az_ulib_capability_command)(az_ulib_model_in model_in, az_ulib_model_out model_out);
 
+/**
+ * @brief       Call a capability in the interface using strings in `az_span`.
+ *
+ * This type defines the signature for the synchronous commands that will be published in an IPC
+ * interface. A synchronous command is the one which runs in the same call stack as the caller.
```

Comment:
>is the one which [](http://example.com/codeflow?start=36&length=16)

 A synchronous command runs in the same call stack as the caller

Reply:
Yes

Reply:
nit^^

Reply:
right, it was just a grammar correction. ".... command runs in the same call stack..."